modules/openglqt/src/openglqtmodule.cpp

```diff
@@ -67,7 +67,8 @@ OpenGLQtModule::OpenGLQtModule(InviwoApplication* app)
 
     // Create GL Context
     CanvasQt::defineDefaultContextFormat();
-    sharedCanvas_ = util::make_unique<CanvasQt>(size2_t(16, 16), "Default");
+
+    sharedCanvas_ = util::make_unique<CanvasQt>(nullptr, size2_t(16, 16), "Default");
```

Comment:
would it not be possible to use the auto QOpenGLContext::globalShareContext(); here instead?

Reply:
Perhaps, if we have a QSurface for it. Have not looked into it but this might be a way to go:
https://doc.qt.io/qt-5/qoffscreensurface.html

Reply:
we don't need any actual fbo or anything here, just a global context.. the canvas is only there since that's usually the only way to get to that context...